cli/config/compose/services/elastic-agent/fleet-server/docker-compose.yml

```diff
@@ -23,5 +23,6 @@ services:
       - "KIBANA_FLEET_HOST=http://kibana:5601"
       - "KIBANA_FLEET_SETUP=${fleetServerMode:-0}"
     platform: ${stackPlatform:-linux/amd64}
+    restart: always
```

Comment:
IIRC this did not work to recreate fleet.

I think we can remove it and rely in the retry that the CI has

Reply:
it did not, I've removed it